example/src/main/java/com/stripe/example/activity/GooglePayPaymentMethodLauncherComposeActivity.kt

```diff
@@ -86,6 +86,7 @@ class GooglePayPaymentMethodLauncherComposeActivity : AppCompatActivity() {
                 googlePayLauncher.present(
                     currencyCode = "EUR",
                     amount = 2500L,
+                    isElements = true, // TODO Define whether this should set `isElements` to `true or `false`
```

Comment:
@samer-stripe I'm proposing a potential lowest common multiple where the decision whether an integration belongs to elements or not can be made. See commit [64ac86](https://github.com/stripe/stripe-android/pull/11395/commits/64ac864c5352d440290c494558bb1bfb623d99dc). Does that surfaces enough to know whether this is an elements integration or not? If so, can you choose the appropriate value for the boolean in each case?

With that said, and if you believe this adds more logic than what's sensible or necessary for clarity, let's consider reducing the level of detail of the integration type.